v1/loader/loader_test.go

```diff
@@ -471,6 +473,51 @@ func TestLoadGuessYAML(t *testing.T) {
 	})
 }
 
+func TestLoadExtension(t *testing.T) {
+	files := map[string]string{
+		"/foo.mock": `{"a": [1,2,3]}`,
+	}
+
+	extension.RegisterExtension(".mock", func(bs []byte, v interface{}) error {
+		return util.UnmarshalJSON(bs, v)
+	})
```

Comment:
[nit] wouldn't that work, too?
```suggestion
	extension.RegisterExtension(".mock", util.UnmarshalJSON)
```

Reply:
I've made the change. Thanks for that catch! The `func(...)` wrapper really wasn't doing anything useful there.